config/int_sp.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I know this is only a test mode key, but it would still be good to not have any keys in the repo.